.github/workflows/pr-review-client-deploy.yml

```diff
@@ -0,0 +1,145 @@
+name: Deploy PR Review Client environment
+
+on:
+  pull_request:
+    branches:
+      - develop  
+    types:
+      - labeled
+      - opened
+      - reopened
+      - synchronize
+
+env:
+  AWS_DEFAULT_REGION: ca-central-1
+  FUNCTION_NAME: "forms-client-pr"
+  IMAGE: forms/client
+  REGISTRY: 687401027353.dkr.ecr.ca-central-1.amazonaws.com/pr_review
+  ROLE_ARN: arn:aws:iam::687401027353:role/forms-lambda-client
+
+jobs:
+  build-and-push-container:
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'Renovate') }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set envs
+        run: echo "PR_NUMBER=$(jq --raw-output .pull_request.number "$GITHUB_EVENT_PATH")" >> $GITHUB_ENV
+
+      - name: Checkout
+        uses: actions/checkout@dc323e67f16fb5f7663d20ff7941f27f5809e9b6 # v2.6.0
+
+      - name: Configure AWS credentials
+        id: aws-creds
+        uses: aws-actions/configure-aws-credentials@67fbcbb121271f7775d2e7715933280b06314838 # v1.7.0
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ca-central-1
+
+      - name: Login to ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@261a7de32bda11ba01f4d75c4ed6caf3739e54be # v1.5.3
+
+      - name: Move dockerignore
+        run: |
+          mv ci/Dockerfile.lambda.dockerignore .
+
+      - name: Build Docker image
+        run: |
+          docker build \
+            --build-arg GIT_SHA=${{ github.sha }} \
+            -t ${{ env.REGISTRY }}/${{ env.IMAGE }}:${{ env.PR_NUMBER }} \
+            -f ci/Dockerfile.lambda .
```

Comment:
We'll need the same build args here that are used to build the ECS task Docker image.